lib/FullScreen.js

```diff
@@ -101,6 +104,14 @@ function (_ImageViewer) {
         this.load(imageSrc, hiResImageSrc);
       } // handle window resize
 
+      // escape listener
+      this._events.escapeListener = (e) => {
```

Comment:
this._events holds the removeListener methods, don't add listener there. See below. This will give error when you do `viewer.destroy()`